.goreleaser.yaml

```diff
@@ -16,28 +16,48 @@ before:
     - go generate ./...
 
 builds:
-  - binary: buildkite-mcp-server
+  - id: buildkite-mcp-server
+    binary: buildkite-mcp-server
     main: ./cmd/buildkite-mcp-server/main.go
     env:
       - CGO_ENABLED=0
     goos:
       - linux
       - windows
       - darwin
-
-dockers:
-  - image_templates:
-    - "buildkite/buildkite-mcp-server:{{ .Version }}"
-    - "buildkite/buildkite-mcp-server:latest"
-    dockerfile: Dockerfile
-    build_flag_templates:
-    - "--pull"
-    - "--label=org.opencontainers.image.created={{.Date}}"
-    - "--label=org.opencontainers.image.title={{.ProjectName}}"
-    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
-    - "--label=org.opencontainers.image.version={{.Version}}"
-    use: buildx
-
+    goarch:
+      - amd64
+      - arm64
+    goamd64:
+      - v2
+    ldflags:
+      - -s -w -X main.version={{.Version}}
+    flags:
+      - -trimpath
+kos:
+  - repositories:
+      - ghcr.io/buildkie/buildkite-mcp-server
+    build: buildkite-mcp-server
+    main: ./cmd/buildkite-mcp-server/
+    creation_time: "{{.CommitTimestamp}}"
+    base_image: 'cgr.dev/chainguard/static:latest@sha256:1ff7590cbc50eaaa917c34b092de0720d307f67d6d795e4f749a0b80a2e95a2c'
```

Comment:
❤️ love all that chainguard stuff!

Good to have something _slightly_ higher level than scratch that [handles stuff like CA bundles](https://images.chainguard.dev/directory/image/static/sbom) for us!